crates/nursery/lib.rs

```diff
@@ -14,16 +12,15 @@ use std::result;
 /// Logs everything to drain `D1`, but in case of it reporting an error,
 /// switching to `D2`. If `D2` returns an error too, `Failover` will return
 /// an error.
-pub struct Failover<D1: Drain, D2: Drain>
-{
+pub struct Failover<D1: Drain, D2: Drain> {
```

Comment:
Technically this formatting change is correct (according to rustfmt), but it's not really relevant to this PR.

Maybe we should clean up the formatting in a separate commit?